airbyte-integrations/bases/connector-acceptance-test/connector_acceptance_test/utils/connector_runner.py

```diff
@@ -160,7 +160,7 @@ def read(cls, container: Container, command: str = None, with_ext: bool = True)
             logging.error(f"Waiting error: {err}, logs: {exception or line}")
             raise
         if exit_status["StatusCode"]:
-            error = exit_status["Error"] or exception or line
+            error = exit_status.get("Error") or exception or line
```

Comment:
Should we update a unit test associated to this so that next time we modify the class, this is taken into account?

Reply:
can you check now?

Reply:
LGTM!